src/components/pill/container.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components"
+import Flex from "src/components/templates/flex"
+import { getColor } from "src/theme"
+
+const getBackground = ({ theme, background, flavour = "neutral", hollow }) => {
+  if (background) return
+  const type = hollow ? "hollow" : "background"
+  const value = getColor(["pill", type, flavour])({ theme })
+  return `background-color: ${value};`
+}
+
+const Container = styled(Flex).attrs(({ round = 999, hollow, flavour, borderColor }) => ({
+  padding: [1, 2],
+  round,
+  border: {
+    side: "all",
+    color: borderColor ? borderColor : ["pill", hollow ? "border" : "background", flavour],
+    size: "1px",
+  },
+  height: "20px",
+  justifyContent: 'center',
+  alignItems: 'center'
```

Comment:
Hmmm, I thought [these](https://www.figma.com/file/28Xv1J95zLbPowJujLuArZ/%F0%9F%93%A6Netdata-Design-System?node-id=6243%3A0) were all the pills.

Reply:
As far as the circular ones, If we remove the `height` and make the `radius` `100%` it will be circular for one digit / char. Let's figure all these out.